Makefile

```diff
@@ -52,7 +52,7 @@ clean:
 
 .PHONY: build
 build:
-	./build.py $(formats)
+	python3 build.py $(formats)
```

Comment:
let's not do this, let the env figure out the right python version to use by adding in the beginning of `build.py` the following shebang:

```
#!/usr/bin/env python
```